### ISSUE TITLE
Codestyle

### ISSUE THREAD

**Issue by silvanmelchior** [roberttidey/userland#2]
Separate code of RaspiMJPEG into multiple files and improve coding style / readability

**Comment by roberttidey** [comment]
Good idea as the one file is too long to make changes easy. I had split off quite few bits into separate functions so should be reasonably simple. I've had a first go at partitioning into RaspiMJPEG.c (main), RaspiMCmds.c RaspiMUtils.c and RaspiMCam.c (MMAL stuff) with a single header file. No actual code changes. Header file defines all functions and shows how they are spread around.

Feel free to move and rationalise further. I'll avoid making further changes for next few days if you are having a go at circular buffer.
